cmd/liqo-controller-manager/main.go

```diff
@@ -339,10 +349,25 @@ func main() {
 	var wg = &sync.WaitGroup{}
 	broadcaster.StartBroadcaster(ctx, wg)
 
-	klog.Info("starting manager as controller manager")
-	if err := mgr.Start(ctx); err != nil {
-		klog.Error(err)
-		os.Exit(1)
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		klog.Info("starting manager as controller manager")
+		if err := mgr.Start(ctx); err != nil {
+			klog.Error(err)
+			os.Exit(1)
+		}
+	}()
+
+	if enableStorage {
+		liqoProvisioner := liqostorageprovisioner.NewLiqoLocalStorageProvisioner(mgr.GetClient(),
+			virtualStorageClassName, storageNamespace, realStorageClassName)
+
+		provisionerName := "liqo.io/storage"
+		provisionController := controller.NewProvisionController(clientset, provisionerName, liqoProvisioner,
+			controller.LeaderElection(false),
+		)
+		provisionController.Run(ctx)
```

Comment:
Would suggest to add this controller to the manager if possible, and keep the previous structure.

Reply:
The provision controller does not implement the Runnable interface, I can wrap it in order to implement this interface (Start(ctx) instead of Run(ctx) method is required)

Do you think this is a good option?

Reply:
Would personally go that route instead of running the manager in a separate goroutine, but that's a matter of taste